platform/branding-api/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,33 @@
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0">
+
+    <bean id="brandingRegistry" class="org.codice.ddf.branding.BrandingRegistryImpl">
+        <cm:managed-properties persistent-id="org.codice.ddf.branding.registry.properties"
```

Comment:
I don't know, I copied this from elsewhere. I left it in because I'm injecting these plugins the same way the other blueprints were (which is now abstracted away). If I don't need this `managed-properties` element, I'll remove it